### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by fi5421**
## Description

This PR adds a health check endpoint to the app. The endpoint returns DB status as well. The endpoint is also documented in Swagger

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN chmod +x /opt/wait-for-it.sh
 COPY ./startup.relational.dev.sh /opt/startup.relational.dev.sh
 RUN chmod +x /opt/startup.relational.dev.sh
 RUN sed -i 's/\r//g' /opt/wait-for-it.sh
-RUN sed -i 's/\r//g' /opt/startup.relational.dev.sh
+# RUN sed -i 's/\r//g' /opt/startup.relational.dev.sh
```
Comment (AwaisKamran) [inline Dockerfile:16]: Why are we commenting this out?

File: Procfile [hunk 2]
```diff
@@ -1,2 +1,3 @@
-web: npm run start:prod
+web: docker compose up
+# web: npm run start:prod
```
Comment (AwaisKamran) [inline Procfile]: Same question here

File: Dockerfile (hunk 1 above)
Comment (fi5421) [reply Dockerfile:16]: The command formatted the bash file to produce an error, commented out temporarily to test docker setup.

File: Procfile (hunk 2 above)
Comment (fi5421) [reply Procfile]: commented out temporarily while testing heroku, fixed in latest commit.

File: Procfile [hunk 3]
```diff
@@ -1,2 +1,3 @@
+web: docker compose up
 web: npm run start:prod
 release: echo '' > .env && npm run migration:run && npm run seed:run:relational
```
Comment (AwaisKamran) [inline Procfile]: Kill this, we dont need it

**Pull request merged**